### PR TITLE
volume scrubber margins and control bar padding issues

### PR DESCRIPTION
**Pull request by wunnath**

### CODE REVIEW

File: sdk/react/controlBar.js [hunk 1]
```diff
@@ -136,15 +137,18 @@ var ControlBar = React.createClass({
...
       },
       fullscreen: {
         onPress: this.onFullscreenPress,
+        iconTouchableStyle: styles.iconTouchable,
         style: [styles.icon, {"fontSize": iconFontSize}, this.props.config.controlBar.iconStyle.active],
         icon: this.props.fullscreen ? this.props.config.icons.compress : this.props.config.icons.expand
       },
       moreOptions: {
         onPress: this.onMorePress,
+        iconTouchableStyle: styles.iconTouchable,
```
Comment (lental) [inline sdk/react/controlBar.js:151]: please add this to all of the other buttons (discovery, share, closedCaption, playPause)

If you want to test it, you can add the buttons to the skin.json

**Comment by lental** [comment]
Talked to Unnath.  Please do not merge until @wunnath  adds new code

File: sdk/react/controlBar.js (hunk 1 above)
Comment (wunnath) [inline sdk/react/controlBar.js:151]: added and tested

File: sdk/react/style/controlBarStyles.json [hunk 2]
```diff
@@ -12,41 +12,27 @@
...
-    "alignItems": "center",
+  },
+  "iconTouchable": {
+    "padding":5,
   },
   "playheadLabel": {
     "flex":0,
-    "paddingLeft":5,
-    "paddingTop":5,
-    "paddingBottom":5,
```
Comment (lental) [inline sdk/react/style/controlBarStyles.json:26]: yes! look at all this red!

**Pull request merged**